charts/lightstepsatellite/values.yaml

```diff
@@ -20,27 +20,56 @@ serviceAccount:
   create: false
   # Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- the name of the service account to use; if not set and create is true, a name is generated using the fullname template
   name:
+  role:
+    # Specifies whether get/watch/list for pods role should be created
+    create: false
+    name: lightstep-pod-reader
+  clusterRole:
+    # Specifies whether get/watch/list for nodes clusterRole should be created
+    create: false
+    name: lightstep-node-reader
+  roleBinding:
+    # Specifies whether role should be created
+    create: false
+    name: lightstep-read-pods
+    # -- if not set and create is true, the `serviceAccount.role.name` is used
+    roleRefName:
+    # -- if not set and and create is true, the generated serviceAccount name is used
+    serviceAccountName:
+  clusterRoleBinding:
+    # Specifies whether clusterRole should be created
+    create: false
+    name: lightstep-read-nodes
+    # -- if not set and create is true, the `serviceAccount.clusterRole.name` is used
+    roleRefName:
+    # -- if not set and and create is true, the generated serviceAccount name is used
+    serviceAccountName:
 
 podAnnotations:
   prometheus.io/scrape: "true"
 
 podSecurityContext: {}
   # fsGroup: 2000
 
+# As of at least version 2021-01-26_23-02-36Z, `drop: ["ALL"]`
+# for securityContext.capabilities results in the lightstep/collector
+# container terminating with the following message:
+#    /bin/sh: 1: exec: /root/collector: Operation not permitted
 securityContext:
-  capabilities:
-    drop:
-      - ALL
+  # capabilities:
+  #   drop:
+  #     - ALL
   readOnlyRootFilesystem: true
-  runAsNonRoot: false
+  runAsNonRoot: true
+  runAsUser: 1000
 
 service:
   type: ClusterIP
   httpPort: 8181
-  grpc: false     #Set to true if you're using GRPC in order to deploy as a headless service for better load balancing
+  # -- set to true if you're using GRPC in order to deploy as a headless service for better load balancing
+  grpc: false
   grpcinsecure: 8184
```

Comment:
This is also hardcoded

Reply:
Same as my comment on the `service.httpPort`